lib/locales/hu.ts

```diff
@@ -0,0 +1,938 @@
+    first_polish_martyrs: 'Saints Benedict, John, Matthew, Isaac and Christian, the First Polish Martyrs',
+    five_wounds_of_the_lord: 'Five Wounds of the Lord',
+    flannan_of_killaloe_bishop: 'Saint Flannan, Bishop',
+    florentius_of_strasbourg_bishop: 'Saint Florentius, Bishop',
+    florian_of_lorch_and_companions_martyrs: 'Saint Florian and Companions, Martyrs',
+    florian_of_lorch_martyr: 'Saint Florian, Martyr',
+    frances_of_rome_religious: 'Saint Frances of Rome, Religious',
+    frances_xavier_cabrini_virgin: 'Saint Frances Xavier Cabrini, Virgin',
+    francis_borgia_priest: 'Saint Francis Borgia, Priest',
+    francis_de_sales_bishop: 'Saint Francis de Sales, Bishop and Doctor of the Church',
+    francis_diaz_del_rincon_priest_and_companions_martyrs: 'Saint Francis Díaz del Rincon, Priest, and Companions, Martyrs',
+    francis_ferdinand_de_capillas_priest: 'Saint Francis Ferdinand de Capillas, Priest and Martyr',*/
+    francis_of_assisi: 'Assisi Szent Ferenc',
+    francis_of_assisi_patron_of_italy: 'Assisi Szent Ferenc, Olaszország patrónusa',
+    /*francis_of_paola_hermit: 'Saint Francis of Paola, Hermit',
+    francis_solanus_priest: 'Saint Francis Solanus, Priest',
+    francis_xavier_priest: 'Saint Francis Xavier, Priest',
+    francis_xavier_seelos_priest: 'Blessed Francis Xavier Seelos, priest',
+    francois_de_montmorency_laval_bishop: 'Saint François de Laval, Bishop',
+    frederic_janssoone_priest: 'Blessed Frédéric Janssoone, Priest',
+    frederic_ozanam_founder: 'Blessed Frédéric Ozanam, Founder',
+    friday_of_the_passion_of_the_lord: 'Friday of the Passion of the Lord',
+    fridolin_of_sackingen_monk: 'Saint Fridolin of Säckingen, Monk',
+    fructuosus_of_braga_martin_of_braga_and_gerald_of_braga_bishops: 'Saints Fructuosus, Martin and Gerald of Braga, Bishops',
+    fructuosus_of_tarragona_bishop_and_augurius_of_tarragona_and_eulogius_of_tarragona_deacons_martyrs: 'Saints Fructuosus, Bishop, Augurius and Eulogius, Deacons, Martyrs',
+    fursa_of_peronne_abbot: 'Saint Fursa, Abbot and Missionary',
+    gabriel_taurin_dufresse_bishop: 'Saint Gabriel-Taurin Dufresse, Bishop and Martyr',
+    gall_of_switzerland_abbot: 'Saint Gall, Abbot and Missionary',
+    genevieve_of_paris_virgin: 'Saint Genevieve, Virgin',
+    george_matulaitis_bishop: 'Blessed George Matulaitis, Bishop',
+    george_of_lydda_martyr: 'Saint George, Martyr',
+    george_of_lydda_martyr_patron_of_england: 'Saint George, Martyr, Patron of England',
+    george_preca_priest: 'Saint George Preca, Priest',
+    gerard_of_csanad_bishop: 'Saint Gerard of Csanád, Bishop and Martyr',
+    germain_of_paris_bishop: 'Saint Germain of Paris, Bishop',
+    germanus_of_auxerre_bishop: 'Saint Germanus of Auxerre, Bishop',
+    gertrude_of_nivelles_abbess: 'Saint Gertrude of Nivelles, Abbess',
+    gertrude_the_great_virgin: 'Saint Gertrude, Virgin',*/
+    gisela_of_hungary: 'Boldog Gizella',
```

Comment:
According to [_Direktórium 2022_](http://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=102), it should be `Boldog Gizella királyné`.